### DISCUSSION TITLE
Parsing erc721 transactions

### DISCUSSION THREAD

**Discussion by LidiaMil** [L-PAY/EVM_Parser#4] [Ideas]
Can we add erc721 parsing functionality?

**Comment by masxxiii** [comment]
Indeed. A need might arise in the future! For the time being parsing native and erc20 transactions suffice. For your info inside the parseBlock( ) function where we create query for FilterLogs call, we can add an extra topic for let's say Approval events from an ERC-721 contracts
`Topics: [][]common.Hash{
			{utils.TransferTopic},
			{utils.ApprovalTopic},
		},`
